packages/template-retail-react-app/package.json

```diff
@@ -52,7 +52,8 @@
         "react-helmet": "^6.1.0",
         "react-hook-form": "^6.15.8",
         "react-intl": "^5.25.1",
-        "react-router-dom": "^5.3.4"
+        "react-router-dom": "^5.3.4",
+        "seedrandom": "^3.0.5"
```

Comment:
don't forget to file 3pp

Reply:
The  `seedrandom` 3PP is already filed and approved via the new [Intake UI]([Intake UI](https://fdxui.sfproxy.buildndeliver.aws-esvc1-useast2.aws.sfdc.cl/secure/saferepo?team=securityhubteam&falcon_service_ids=f88c3ce7-35d5-4dcb-aafc-dcb5351cb89f&showSafeRepo=1&services=security-hub).